feature/qos/feature.textproto

```diff
@@ -63,33 +84,60 @@ telemetry_path {
 config_path {
   path: "/qos/scheduler-policies/scheduler-policy/config/name"
 }
+telemetry_path {
+  path: "/qos/scheduler-policies/scheduler-policy/state/name"
+}
 config_path {
   path: "/qos/scheduler-policies/scheduler-policy/schedulers/scheduler/config/priority"
 }
+telemetry_path {
+  path: "/qos/scheduler-policies/scheduler-policy/schedulers/scheduler/state/priority"
+}
 config_path {
   path: "/qos/scheduler-policies/scheduler-policy/schedulers/scheduler/config/sequence"
 }
+telemetry_path {
+  path: "/qos/scheduler-policies/scheduler-policy/schedulers/scheduler/state/sequence"
+}
 config_path {
   path: "/qos/scheduler-policies/scheduler-policy/schedulers/scheduler/config/type"
 }
+telemetry_path {
+  path: "/qos/scheduler-policies/scheduler-policy/schedulers/scheduler/state/type"
+}
 config_path {
   path: "/qos/scheduler-policies/scheduler-policy/schedulers/scheduler/inputs/input/config/id"
 }
+telemetry_path {
+  path: "/qos/scheduler-policies/scheduler-policy/schedulers/scheduler/inputs/input/state/id"
+}
 config_path {
   path: "/qos/scheduler-policies/scheduler-policy/schedulers/scheduler/inputs/input/config/input-type"
 }
+telemetry_path {
+  path: "/qos/scheduler-policies/scheduler-policy/schedulers/scheduler/inputs/input/state/input-type"
+}
 config_path {
   path: "/qos/scheduler-policies/scheduler-policy/schedulers/scheduler/inputs/input/config/queue"
 }
+telemetry_path {
+  path: "/qos/scheduler-policies/scheduler-policy/schedulers/scheduler/inputs/input/state/queue"
+}
 config_path {
   path: "/qos/scheduler-policies/scheduler-policy/schedulers/scheduler/inputs/input/config/weight"
 }
 telemetry_path {
-  path: "/qos/scheduler-policies/scheduler-policy/schedulers/scheduler/inputs/input/state/queue"
```

Comment:
ah nice catch, looks like this was a duplicate

Reply:
Yes I wrote script in /featureprofiles/tools/validate_paths.go to check for missing paths. I will open source the code so that it runs automatically in the backend.